tests/ecosystem/deployment/test_ocs_basic_install.py

```diff
@@ -177,6 +202,31 @@ def test_deployment(self):
         log.info(f"Waiting {wait_time} seconds...")
         time.sleep(wait_time)
 
+        # Create MDS pods for CephFileSystem
+
+        self.fs_data = defaults.CEPHFILESYSTEM_DICT.copy()
+        self.fs_data['metadata']['namespace'] = ENV_DATA['cluster_namespace']
+
+        global CEPH_OBJ
+        CEPH_OBJ = OCS(**self.fs_data)
+        CEPH_OBJ.create()
+        assert POD.wait_for_resource(
+            condition='Running', selector='app=rook-ceph-mds',
+            resource_count=2, timeout=600
+        )
+
+        # Check for CephFilesystem creation in ocp
+        cfs_data = CFS.get()
+        cfs_name = cfs_data['items'][0]['metadata']['name']
+
+        # Check successful creation of cephfilesystem from ceph backend
+        ct_pod = pod.get_ceph_tools_pod()
+        cmd_out = ct_pod.exec_ceph_cmd(ceph_cmd="ceph fs ls ")
+
+        for pool in cmd_out:
```

Comment:
@nehaberry I'm not sure if I understand what is the output.
in cmd_out we should expect to get all pools in ceph right?
In that case, the code can be switched to:
```python
if cfs_name in [pool.get('name') for pool in cmd_out]:
    log.info()
```

If that not the case, what is the output?

Reply:
@RazTamir yeah the loop will traverse through the pool list and only when the pool name matches the cfs_name, it will print the success method. 

It seems both the loops are doing similar thing, do you suggest me to try with the code above ? 

I will work on it

Reply:
Due to max VPC issue, unable to re-test code with your suggested code.

Reply:
What issue?
This code is identical to yours but in a more pytonic way (list comprehension)

Reply:
> What issue?
 Max VPC count in AWS, hence could not test fresh install

> This code is identical to yours but in a more pytonic way (list comprehension)
sure. thanks